main.tex

```diff
@@ -1563,25 +1558,23 @@
 
 \begin{enumerate}[label=(\arabic*)]
 
-\item  An Owner or Occupier of a Lot must ensure that music or noise from their Lot to Residential Lots in the Building at all times complies with the applicable law at any point in time.
+\item  An Owner or Occupier of a Lot must ensure that music or noise from their Lot to Residential Lots in the Building at all times complies with the applicable Law at any point in time.
 
 \end{enumerate}
 
 \item  Pest Control
 
 \begin{enumerate}[label=(\arabic*)]
 
-\item  An Owner or Occupier of a Lot must, at their cost, keep the Lot free from pests, including (without limitation) cockroaches, spiders, ants, rodents and other vermin.
+\item  An Owner or Occupier of a Lot must, at their cost, keep the Lot free from Animal Pests.
 
 \end{enumerate}
 
 \item  Temperature Control
 
 \begin{enumerate}[label=(\arabic*)]
 
-\item  An Owner or Occupier of a Lot must not use chemicals, burning fluids or alcohol in lighting or heating the Lot without the prior written consent of the Owners Corporation.
-
-\item  An Owner or Occupier of a Lot must obtain the prior written consent of the Owners before the Lot Owner uses any services or equipment for cooling, heating, lighting or circulating air in the Lot other than those that the Owners Corporation supplies, if any.
```

Comment:
Did you get the written permission of the Owners Corporation for your lamp? Your air purifier? Your candle?